python/cudf/cudf_pandas_tests/third_party_integration_tests/ci/ci_run_library_tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cleanup() {
+    rm tests/results-*.pickle
+}
+
+trap cleanup EXIT
+
+runtest_gold() {
+    local lib=$1
+    local test_keys=${@:2}
+
+    pytest \
+    -v \
+    --continue-on-collection-errors \
+    --cache-clear \
+    --junitxml="${RAPIDS_TESTS_DIR}/junit-${lib}-gold.xml" \
+    --numprocesses=${NUM_PROCESSES} \
+    --dist=worksteal \
+    ${TEST_DIR}/test_${lib}*.py \
+    ${test_keys}
+}
+
+runtest_cudf_pandas() {
+    local lib=$1
+    local test_keys=${@:2}
+
+    pytest \
+    -p cudf.pandas \
+    -v \
+    --continue-on-collection-errors \
+    --cache-clear \
+    --junitxml="${RAPIDS_TESTS_DIR}/junit-${lib}-cudf-pandas.xml" \
+    --numprocesses=${NUM_PROCESSES} \
+    --dist=worksteal \
+    ${TEST_DIR}/test_${lib}*.py \
+    ${test_keys}
+}
```

Comment:
Minor nit: could we combine these functions using a boolean parameter to indicate whether or not to run with cudf.pandas? If you find it challenging, don't worry about it. We can always revisit in a follow-up.

Reply:
Okay I’ll follow-up with a PR that combines these two functions